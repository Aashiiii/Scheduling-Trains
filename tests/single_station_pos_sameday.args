Waverton Gymea "08:52:39"
