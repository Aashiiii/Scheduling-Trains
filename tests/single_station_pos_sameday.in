Waverton::Gymea::08:54:27
Waverton::Gymea::08:24:23
Waverton::Gymea::08:52:43
Waverton::Gymea::03:45:43
