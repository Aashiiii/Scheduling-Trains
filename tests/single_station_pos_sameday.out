The next train to Gymea from Waverton departs at 08:52:43
